.github/workflows/macos-ci.yml

```diff
@@ -46,6 +46,7 @@ jobs:
           brew list ccache || brew install ccache
           brew list pkg-config || brew install pkg-config
           brew list glfw3 || brew install glfw3
+          brew list libuv || brew install libuv
```

Comment:
Is this a new dependency or just a missing one and loaded implicitly by some other?

Reply:
It's implicitly loaded due to `node-loop`, which is built as part of the whole package.

Reply:
Node dependencies should not be built as part of macOS build.